config/database.yml

```diff
@@ -0,0 +1,16 @@
+development:
+  adapter: postgresql
+  encoding: unicode
+  database: cake_master_development
+  pool: 5
+  username: pguser
+  password: pguser
```

Comment:
!! We never commit database.yml to repo - please remove it from repo (but from local drive) and change your credentials in database. http://www.michaelrigart.be/en/blog/dont-commit-sensitive-data.html